include/igl/tri_tri_intersect.h

```diff
@@ -5,6 +5,61 @@
 // This Source Code Form is subject to the terms of the Mozilla Public License 
 // v. 2.0. If a copy of the MPL was not distributed with this file, You can 
 // obtain one at http://mozilla.org/MPL/2.0/
+
+/*
+*  
+*  C++ version based on the routines published in    
```

Comment:
Might be worth mentioning that this code is under MIT license, according to [this repository](https://github.com/erich666/jgt-code) (and the original code can be grabbed from [this folder](https://github.com/erich666/jgt-code/tree/master/Volume_08/Number_1/Guigue2003), since the original web page shows a 404 error).

Reply:
I added a reference to the original repository and copy of the original license.